include/groov/config.hpp

```diff
@@ -52,9 +54,8 @@ template <typename C, stdx::ct_string Name>
 using get_child = typename C::template child_t<Name>;
 
 template <typename T>
-concept fieldlike = named<T> and containerlike<T> and
-                    std::unsigned_integral<decltype(T::mask)> and
```

Comment:
question: why remove this requirement?

Reply:
`mask` is now a template... and if it is templated here, it crashes clang :)
There are a few other things to put in here now too, maybe can strengthen it later.